src/OpenTelemetry/README.md

```diff
@@ -243,6 +245,16 @@ using var tracerProvider = Sdk.CreateTracerProviderBuilder()
     .Build();
 ```
 
+It is also to configure the resource by using `AddEnvironmentVariableDetector`
+together with following environmental variables:
+
+<!-- markdownlint-disable MD013 -->
+| Environment variable       | Description                                        |
+| -------------------------- | -------------------------------------------------- |
+| `OTEL_RESOURCE_ATTRIBUTES` | Key-value pairs to be used as resource attributes. |
```

Comment:
Perhaps link add a link to the spec (permalink to the version currently implemented).

Reply:
https://github.com/open-telemetry/opentelemetry-dotnet/pull/2209/commits/d5198652166c11a4ae5cfb893b466094eba8dffe